src/main/java/com/actiontech/dble/cluster/general/AbstractConsulSender.java

```diff
@@ -157,5 +164,26 @@ protected void firstReturnToCluster() throws Exception {
         }
     }
 
+    public List<String> fetchRenewThread() {
+        String onlineStr = ClusterPathUtil.getOnlinePath(SystemConfig.getInstance().getInstanceName());
+        List<String> renewThread = Lists.newArrayList();
+        lockMap.values().stream().
+                filter(c -> !c.getName().endsWith(onlineStr) && c.isAlive()).
+                forEach(c -> renewThread.add(c.getName()));
```

Comment:
use .collect(toList()) instead is better

Reply:
ok